.circleci/config.yml

```diff
@@ -232,7 +231,9 @@ jobs:
             - images/3scaleoperator-image.tar
 
   tag-operator-image-master:
-    machine: true
+    machine:
```

Comment:
Obtained expected / available values from here:
https://circleci.com/docs/2.0/configuration-reference/#machine

It seems that the old default was circleci/classic:latest so I set the image to that